src/main/java/com/toptrumps/cli/CommandLineUI.java

```diff
@@ -141,4 +143,11 @@ public void showActivePlayer(){
         System.out.println("The active player is: " + activePlayerName);
     }
 
+    public void showRound(){
+        String round = "Round " + game.getRoundCounter();
+        System.out.println(round);
```

Comment:
I know someone will comment that it does the same thing twice, but it's how it is in the example CLI output. We can change it later.

Reply:
OK